roles/olm_operator/tasks/main.yml

```diff
@@ -55,10 +55,25 @@
     - operator_packagemanifest is defined
     - operator_packagemanifest.resources is defined
     - operator_packagemanifest.resources | length
-  failed_when: operator_packagemanifest is skipped
+  failed_when: false
+  ignore_errors: true
```

Comment:
do we really need failed_when and ignore_errors together? Maybe `ignore_errors` is enough?

Reply:
> do we really need failed_when and ignore_errors together? Maybe `ignore_errors` is enough?

Yeah, hehe I was attempting an ignore retry :) Thanks!